api/lib/schema.js

```diff
@@ -72,6 +72,8 @@ const typeDefs = gql`
     id: String
     voter: NetworkAccount
     option: String
+    amount: Coin # Substrate only
+    time: String # Substrate only
```

Comment:
```suggestion
```

Reply:
Yes, let's remove this. In the end we cannot know in any network :shrug: